cdc/model/sink.go

```diff
@@ -719,6 +765,112 @@ func (t *SingleTableTxn) GetCommitTs() uint64 {
 	return t.CommitTs
 }
 
+// TrySplitAndSortUpdateEvent split update events if unique key is updated
+func (t *SingleTableTxn) TrySplitAndSortUpdateEvent() error {
+	if len(t.Rows) < 2 {
+		return nil
+	}
+	newRows, split, err := trySplitAndSortUpdateEvent(t.Rows)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	// some updated events is split, need to sort
+	if split {
```

Comment:
This logic can looks can be put inside the `trySplitAndSortUpdateEvent`, and `split` no need to be returned

Reply:
fixed